### PR TITLE
Add support for healthcheck in compose v3

### PR DESCRIPTION
**Pull request by vdemeester**
This would add `healthcheck` support in a simple manner:

```yaml
version: "3"

services:
  foo:
    image: busybox
    command: top
    healthcheck:
      command: cat /etc/password
      interval: 10s
      timeout: 2s
      retries: 3
```

/cc @aanand @dnephin 

🐸

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

### CODE REVIEW

File: schema/data/config_schema_v3.json [hunk 1]
```diff
@@ -101,6 +101,21 @@
 
         "external_links": {"type": "array", "items": {"type": "string"}, "uniqueItems": true},
         "extra_hosts": {"$ref": "#/definitions/list_or_dict"},
+        "healthcheck": {
+          "type": "object",
+          "properties": {
+            "interval": {"type":"string"},
+            "timeout": {"type":"string"},
+            "retries": {"type": "number"},
+	    "command": {
```
Comment (dnephin) [inline schema/data/config_schema_v3.json]: indentation is off here?

Comment (vdemeester) [inline schema/data/config_schema_v3.json]: it looks like it :sweat:

File: types/types.go [hunk 2]
```diff
@@ -84,6 +84,7 @@ type ServiceConfig struct {
 	ExternalLinks   []string          `mapstructure:"external_links"`
 	ExtraHosts      map[string]string `mapstructure:"extra_hosts" compose:"list_or_dict_colon"`
 	Hostname        string
+	HealthCheck     HealthCheckConfig
```
Comment (vdemeester) [inline types/types.go]: Hum, good point :)

**Comment by vdemeester** [comment]
Build fixed 👼

**Pull request merged**